ui/component/channelAbout/view.jsx

```diff
@@ -54,10 +57,41 @@ function ChannelContent(props: Props) {
           <div className="media__info-text">
             <ClaimTags uri={uri} type="large" />
           </div>
+
+          <label>{__('Total Publishes')}</label>
+          <div className="media__info-text">{claim.meta.claims_in_channel}</div>
+
+          <label>{__('Last Updated')}</label>
+          <div className="media__info-text">
+            <DateTime timeAgo uri={uri} />
+          </div>
+
+          <label>{__('Claim ID')}</label>
+          <div className="media__info-text">
+            <div className="media__info-text media__info-text--constrained">{claim.claim_id}</div>
+          </div>
+
+          <label>{__('Staked LBC')}</label>
+          <div className="media__info-text">
+            <CreditAmount
+              badge={false}
+              amount={parseFloat(claim.amount) + parseFloat(claim.meta.support_amount)}
+              precision={8}
+            />{' '}
+            (
+            <Button
+              button="link"
+              label={__('view all claims', {
+                normalized_uri: uri,
+              })}
+              navigate={`/$/${PAGES.TOP}?name=${claim.name}`}
```

Comment:
I don't think this link does what you want. It shows all claims for that claim name, which will just be other channels with the same name?
http://lbry.tv/$/top?name=@kauffj